src/main/kotlin/com/personio/synthetics/builder/RetryOptionsBuilder.kt

```diff
@@ -0,0 +1,15 @@
+package com.personio.synthetics.builder
+
+import com.datadog.api.client.v1.model.SyntheticsTestOptionsRetry
+
+/**
+ * Builds the configuration for the step retry behavior
+ */
+class RetryOptionsBuilder {
+    var count = 0L
+    var intervalMs = 0.0
+
+    fun build(): SyntheticsTestOptionsRetry {
+        return SyntheticsTestOptionsRetry().count(count).interval(intervalMs)
```

Comment:
`count` and `interval` values have restrictions:
- Retry count must be between 0 and 5.
- Retry interval must be between 0 and 5000.

Could you please add them to the function. Example in browser tests - https://github.com/personio/datadog-synthetic-test-support/blob/master/src/main/kotlin/com/personio/synthetics/config/BrowserTestConfig.kt#L89